src/main/java/com/epam/service/PersonService.java

```diff
@@ -24,4 +24,23 @@ public Person getPerson(String name){
         return null;
     }
 
+    public boolean addPersonToDataBase(Person person){
+        return personDAO.addPersonToDataBase(person);
+    }
+
+    public boolean updatePassword(String email, String password) {
```

Comment:
Whats happened, if i pass empty string here?

Reply:
I think that this case will not happen ever cuz when user wana to change password, we ll read all neccesary data from him (current user), but i added if-else constructor to method, wich instantly return false, if password || email == ""; But, in my opinion that is unnecessary cuz, we must check if password matches out password pattern(as i did in registration controller).

Reply:
but if i want to use your method from another service, and i will pass null here